### ISSUE TITLE
Check whether we can enable web inspector by default on older Qt versions

### ISSUE THREAD

**Issue by The-Compiler** [qutebrowser/qutebrowser#4728]
On Qt < 5.11, `:inspector` requires the `--enable-webengine-inspector` flag because it listens on a local port, and I suspect the inspector could be accessed remotely: https://bou.ke/blog/hacking-developers/ (even without DNS rebinding, it's likely possible to execute commands).

However, we should check whether that's actually possible - if not, we might consider lifting that restriction, or exposing it in a more permanent way (e.g. a setting).

Another possibility would be letting it run [on a port which is blocked by Chromium](https://superuser.com/questions/188058/which-ports-are-considered-unsafe-by-chrome) - here are possible ports we could (ab)use:

```
    2049,    // nfs
    3659,    // apple-sasl / PasswordServer
    4045,    // lockd
    6000,    // X11
    6665,    // Alternate IRC [Apple addition]
    6666,    // Alternate IRC [Apple addition]
    6667,    // Standard IRC [Apple addition]
    6668,    // Alternate IRC [Apple addition]
    6669,    // Alternate IRC [Apple addition]
```

**Comment by The-Compiler** [comment]
Superseded by #3839
